### ISSUE TITLE
How can I use `lineman serve` with basic auth?

### ISSUE THREAD

**Issue by mark-rushakoff** [linemanjs/lineman#341]
Does lineman have a built-in way to start the server behind a simple basic auth server? My use case is that we have a development server facing the public internet, and while we don't want the app to be accessible to the internet, we can accept basic auth as "good enough" for this case.

If there isn't a way to do it already, would you accept a pull request?

**Comment by searls** [comment]
This won't help your problem, but @jasonkarns and I did this once in a roundabout way on a client project. I believe in that case all we did was set up Basic Auth on our API server (to which we were proxying requests to anyway).

Otherwise, I'd start looking into Express 3's options (or related modules) that enable basic auth. It's probably as simple as finding that and then wrapping your `app` in it in your `config/server.js`.
